### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by schnf8**
## Additions
- Added comments to all AppRessources entries
- Replaced hardcoded strings with AppRessources entries

### CODE REVIEW

File: BFH_USZ_PICC/BFH_USZ_PICC/Shell.cs [hunk 1]
```diff
@@ -111,7 +111,7 @@ protected override void OnAppearing()
                 var masterData = await AskUserToAddMasterData();
                 if (masterData)
                 {
-                    await DeepNavigateAsync(MenuItemKey.Settings, typeof(UserMasterDataPage), new List<object> { UserMasterDataPageDisplayMode.Edit });
+                    await DeepNavigateAsync(MenuItemKey.Settings, typeof(UserMasterDataPage), new List<object> { true });
```
Comment (swisszeni) [inline BFH_USZ_PICC/BFH_USZ_PICC/Shell.cs:114]: What is the meaning of true?

File: BFH_USZ_PICC/BFH_USZ_PICC/Models/UserMasterData.cs [hunk 2]
```diff
@@ -5,8 +5,8 @@
 using System.Threading.Tasks;
 
 namespace BFH_USZ_PICC.Models
-{
-    public enum Sex
+{ 
+    public enum Salutation
```
Comment (swisszeni) [inline BFH_USZ_PICC/BFH_USZ_PICC/Models/UserMasterData.cs:9]: Why not Sex?

File: BFH_USZ_PICC/BFH_USZ_PICC/Views/SettingsPage.xaml.cs [hunk 3]
```diff
@@ -50,7 +50,7 @@ void AddNotificationButtonClicked(object o, EventArgs e)
 
         void PersonalMasterDataClicked(object o, EventArgs e)
         {
-            Navigation.PushAsync(new BasePage(typeof(UserMasterDataPage), new List<object> { UserMasterDataPageDisplayMode.View }));
+            Navigation.PushAsync(new BasePage(typeof(UserMasterDataPage), new List<object> { false }));
```
Comment (swisszeni) [inline BFH_USZ_PICC/BFH_USZ_PICC/Views/SettingsPage.xaml.cs:53]: What is the meaning of false?

File: BFH_USZ_PICC/BFH_USZ_PICC/Models/UserMasterData.cs (hunk 2 above)
Comment (schnf8) [reply BFH_USZ_PICC/BFH_USZ_PICC/Models/UserMasterData.cs:9]: We could call it "Sex" but on the related page, it is also called "Salutation"

File: BFH_USZ_PICC/BFH_USZ_PICC/Views/SettingsPage.xaml.cs (hunk 3 above)
Comment (schnf8) [reply BFH_USZ_PICC/BFH_USZ_PICC/Views/SettingsPage.xaml.cs:53]: false = UserMasterDetail page won't be opened in "enable" mode (user needs to press the "Edit" button before he/she can edit the personal data)

File: BFH_USZ_PICC/BFH_USZ_PICC/Shell.cs (hunk 1 above)
Comment (schnf8) [reply BFH_USZ_PICC/BFH_USZ_PICC/Shell.cs:114]: true = UserMasterDetail page will be opened in "enable" mode

**Pull request merged**